tests/providers/test_lorem.py

```diff
@@ -4,6 +4,8 @@
 
 from faker.providers.lorem.az_AZ import Provider as AzAzLoremProvider
 from faker.providers.lorem.cs_CZ import Provider as CsCzLoremProvider
+from faker.providers.lorem.de_DE import Provider as DeDeLoremProvider
+from faker.providers.lorem.de_AT import Provider as DeAtLoremProvider
```

Comment:
these need to be sorted alphabetically. You can run `make lint` to have [`isort`](https://pycqa.github.io/isort/) fix it for you.

Reply:
Sorry for my bad formatting skills, but now I ran `make lint` and added the changes in [e56ac0c](https://github.com/joke2k/faker/pull/1610/commits/e56ac0c0625b83f62940ff161ca2b64045b71ad9)